ur_ros2_control_demos/launch/ur5e_run_move_group.launch.py

```diff
@@ -28,26 +30,76 @@ def load_yaml(package_name, file_path):
+    physical_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                   robot_name, 'physical_parameters.yaml')
+    visual_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                 robot_name, 'visual_parameters.yaml')
+
+    # common parameters
+    # If True, enable the safety limits controller
+    safety_limits = False
+    # The lower/upper limits in the safety controller
+    safety_pos_margin = 0.15
+    # Used to set k position in the safety controller
+    safety_k_position = 20
+
+    use_ros2_control = True
+
+    script_filename = os.path.join(
+        get_package_share_directory('ur_robot_driver'),
+        'resources',
+        'ros_control.urscript')
+
+    input_recipe_filename = os.path.join(
         get_package_share_directory('ur_robot_driver'),
         'resources',
-        'ur5.urdf.xacro')
-    robot_description_config = xacro.process_file(robot_description_path)
+        'rtde_input_recipe.txt')
+
+    output_recipe_filename = os.path.join(
+        get_package_share_directory('ur_robot_driver'),
+        'resources',
+        'rtde_output_recipe.txt')
+
+    # Get URDF via xacro
+    robot_description_path = os.path.join(get_package_share_directory('ur_description'), 'urdf', 'ur.xacro')
+
+    robot_description_config = xacro.process_file(robot_description_path,
+                                                  mappings={'joint_limit_params': joint_limits_params,
+                                                            'kinematics_params': kinematics_params,
+                                                            'physical_params': physical_params,
+                                                            'visual_params': visual_params,
+                                                            'safety_limits': str(safety_limits).lower(),
+                                                            'safety_pos_margin': str(safety_pos_margin),
+                                                            'safety_k_position': str(safety_k_position),
+                                                            'use_ros2_control': str(use_ros2_control).lower(),
+                                                            'script_filename': script_filename,
+                                                            'input_recipe_filename': input_recipe_filename,
+                                                            'output_recipe_filename': output_recipe_filename,
+                                                            'robot_ip': '10.0.1.186'}
+                                                  )
+
     robot_description = {'robot_description': robot_description_config.toxml()}
 
-    robot_description_semantic_config = load_file('ur5_moveit_config', 'config/ur5.srdf')
+    robot_description_semantic_config = load_file(robot_name + '_moveit_config', 'config/' + robot_name + '.srdf')
```

Comment:
Is this a reason to rename `ur5_e` to `ur5`?

Reply:
No, the rename was a mistake. This is handled now in different manner by replacing potential '_' with ''. The idea is to have launch.py files as generic as possible.